api/v3/Case.php

```diff
@@ -210,9 +210,19 @@ function civicrm_api3_case_get($params) {
   //search by client
   if (!empty($params['contact_id'])) {
     $ids = array();
+
+    $statusId = NULL;
```

Comment:
Status ID is provided in the form of Option Name while we need the Option Value.